### PR TITLE
Document intro + logo

### PR DESCRIPTION
**Pull request by RuiAAPeres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+</p>
+
+
+iOS Playbook 📚
+==================================
+
+At babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. The iOS's playbook embodies that value by giving an overview on how we work:
+
+1. Who's part of the team.
+2. Which squad each individual belongs to and availability.
```
[the remark this answers is not in the document]
Comment (wltrup) [reply README.md]: Then I think it should be "Which squad each individual belongs to, and its availability."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+| David Rodrigues         | [@dmcrodrigues](https://twitter.com/dmcrodrigues)             |
+| Diego Petrucci          | [@diegopetrucci](https://twitter.com/diegopetrucci)           |
+| Ilya Puchka             | [@ilyapuchka](https://twitter.com/ilyapuchka)                 |
+| Jason Dobo              |                                                               |
+| Martin Nygren           |                                                               |
+| Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
+| Rui Peres               | [@peres](https://twitter.com/peres)                           |
+| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
+| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
+| Wagner Truppel          |                                                               |
```
Comment (wltrup) [inline README.md]: `[@wltrup](https://github.com/wltrup)`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+| Rui Peres               | [@peres](https://twitter.com/peres)                           |
+| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
+| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
+| Wagner Truppel          |                                                               |    
+| Witold Skibniewski      | [@mr_witold](https://twitter.com/mr_witold)                   |
+
+
+## 2. Which squad each individual belongs to and availability
+
+By definition members work on their respective squad, although they are free to work in different squads if the work load justifies it.
```
Comment (wltrup) [inline README.md]: "By definition, members work on their respective squads, although..."

File: README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+| GP at Hand                    | Diego                    |    1/1       |
+| Project Avalon                | Sergey                   |    1/1       |
+| Project Minkowski             | Anders                   |    1/2       |
+| Project Bayes                 | Michael                  |    1/1       |
+
+## 3. OSS maitained projects
+
+| Project name                  | Owner(s)                 | Stars        |  
+|-------------------------------|--------------------------| ------------ |
+| Bento                         | Anders, David, Serg      | [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/Bento.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/Bento/stargazers/)             |
```
Comment (wltrup) [inline README.md]: Sergey, not Serg.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+| DrawerKit                     | David, Wagner            |    [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/DrawerKit.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/DrawerKit/stargazers/)       |
+| ReactiveFeedback              | Anders, Serg             |    [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/ReactiveFeedback.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/ReactiveFeedback/stargazers/)        |
```
Comment (wltrup) [inline README.md]: Sergey, not Serg.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+## 4. OSS maintained projects
+
+// Working in progress ⚠️
```
Comment (wltrup) [inline README.md]: Work in progress, not working in progress.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+## 5. Technical documents
+
+// Working in progress ⚠️
```
Comment (wltrup) [inline README.md]: Ditto.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+## 6. New hires checklist
+
+// Working in progress ⚠️
```
Comment (wltrup) [inline README.md]: Ditto.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+## 7. Interview process
+
+// Working in progress ⚠️
```
Comment (wltrup) [inline README.md]: Ditto.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+## 8. Code guidelines, standards and etiquette
+
+// Working in progress ⚠️
```
Comment (wltrup) [inline README.md]: Ditto.

File: README.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+| Martin Nygren           |                                                               |
+| Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
+| Rui Peres               | [@peres](https://twitter.com/peres)                           |
+| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
+| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
+| Wagner Truppel          |                                                               |    
+| Witold Skibniewski      | [@mr_witold](https://twitter.com/mr_witold)                   |
+
+
+## 2. Which squad each individual belongs to and availability
```
Comment (wltrup) [inline README.md]: "Which squad each individual belongs to, and its availability."

File: README.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+
+By definition members work on their respective squad, although they are free to work in different squads if the work load justifies it.
+
+
+| Squad Name                    | Members                  | Availability |
```
Comment (wltrup) [inline README.md]: I don't understand what the availability column is saying.

File: README.md [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+| Consultation Experience       | Ilya                     |    1/1       |
+| Healthcheck                   | Ben, Wagner              |    2/2       |
+| Core                          | Witold, Jason            |    2/4       |
+| Professional Services         | Danilo                   |    1/1       |
+| GP at Hand                    | Diego                    |    1/1       |
+| Project Avalon                | Sergey                   |    1/1       |
+| Project Minkowski             | Anders                   |    1/2       |
+| Project Bayes                 | Michael                  |    1/1       |
+
+## 3. OSS maitained projects
```
Comment (mluisbrown) [inline README.md]:
```suggestion
## 3. OSS maintained projects
```

File: README.md [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+
+| Project name                  | Owner(s)                 | Stars        |  
+|-------------------------------|--------------------------| ------------ |
+| Bento                         | Anders, David, Serg      | [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/Bento.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/Bento/stargazers/)             |
+| DrawerKit                     | David, Wagner            |    [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/DrawerKit.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/DrawerKit/stargazers/)       |
+| ReactiveFeedback              | Anders, Serg             |    [![GitHub stars](https://img.shields.io/github/stars/BabylonPartners/ReactiveFeedback.svg?style=social&label=Star&maxAge=2592000)](https://GitHub.com/BabylonPartners/ReactiveFeedback/stargazers/)        |
+
+## 4. OSS maintained projects
```
Comment (mluisbrown) [inline README.md]: This is a repeat of section 3, so it should be removed. And the following section numbers adjusted accordingly.

File: README.md [hunk 15]
```diff
@@ -0,0 +1,85 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+iOS Playbook 📚
+==================================
+
+At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview on how we work:
```
Comment (diegopetrucci) [inline README.md]:
```
At Babylon, **transparency** is a core value that we believe should be present in everything we do. This playbook embodies it by giving an overview of how we work:
```

Comment (diegopetrucci) [inline README.md]:
```suggestion
At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview on how we work:
```
```suggestion
```
At Babylon, **transparency** is a core value that we believe should be present in everything we do. This playbook embodies it by giving an overview of how we work:
```
```

Comment (diegopetrucci) [inline README.md]:
```suggestion
```
At Babylon, **transparency** is a core value that we believe should be present in everything we do. This playbook embodies it by giving an overview of how we work:
```
```

Comment (diegopetrucci) [inline README.md]:
```suggestion
At Babylon, **transparency** is a core value that we believe should be present in everything we do. This playbook embodies it by giving an overview of how we work:
```

Comment (diegopetrucci) [reply README.md]: Either transparency is a core value or it isn't. If it is, you try/believe to make everything as transparent as possible, if you don't, you don't. Believing in something, IMO, does not correlate on acting on it, so we should state that transparency *is* a value, not a hypothetical.

Comment (diegopetrucci) [reply README.md]: The `of` was suggested by Grammarly, and I think it makes sense.

Comment (RuiAAPeres) [reply README.md]: I am sorry, but I don't fully agree with this suggestion, although I am not a native speaker. This sentence was taken from the iOS Interview Repo, as is, without any change from me.

File: README.md (hunk 12 above)
Comment (RuiAAPeres) [reply README.md]: Availability for new members to join.

File: README.md (hunk 15 above)
Comment (mluisbrown) [reply README.md]: As a native speaker, it should be "overview *of*". As for the other change...meh. I would be inclined to leave it as it is.

Comment (diegopetrucci) [reply README.md]: 👍

File: README.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+
+
+iOS Playbook 📚
+==================================
+
+At Babylon, we firmly believe that **transparency** is a core value that should be present in everything we do. This playbook embodies that value by giving an overview of how we work:
+
+1. Who's in the team.
+2. Which squad each individual belongs to, and its availability.
+3. OSS maintained projects.
```
Comment (wltrup) [inline README.md]: OSS-maintained  (note the hyphen)

File: README.md (hunk 13 above)
Comment (wltrup) [reply README.md]: Actually, it should have an hyphen too: "OSS-maintained"

File: README.md (hunk 16 above)
Comment (wltrup) [reply README.md]: As per your request for the rule, @RuiAAPeres, check the "noun+participate" case in https://en.oxforddictionaries.com/punctuation/hyphen.

**Pull request merged**